plugins/provisioning-terraform-gcp-repository-ar/src/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { PluginInstallation } from "@amplication/code-gen-types";
+import { name as PackageName } from "../package.json";
+import { Settings } from "./types";
+import defaultSettings from "../.amplicationrc.json";
+
+export const getPluginSettings = (
+  pluginInstallations: PluginInstallation[]
+): Settings => {
+  const plugin = pluginInstallations.find(
+    (plugin) => plugin.npm === PackageName
+  );
+
+  const userSettings = plugin?.settings ?? {};
+
+  const settings: Settings = {
+    ...defaultSettings.settings,
+    ...userSettings,
+  };
+
+  return settings;
+};
+
+export const getTerraformDirectory = (
+  pluginInstallations: PluginInstallation[],
+  serverBaseDirectory: string
+): string => {
+  const plugin = pluginInstallations.find(
+    (plugin) =>
+      plugin.npm === "@amplication/plugin-provisioning-terraform-gcp-core"
+  );
+
+  if (!plugin) {
+    throw new Error(
+      "TerraformGcpRepositoryArPlugin: is dependent on 'Terraform - GCP Core' plugin"
```

Comment:
Would it make more sense to use the plugin name here instead of the class name? `TerraformGcpRepositoryArPlugin`

You can import this from `package.json`